### PR TITLE
Clarification of the config

### PR DESCRIPTION
**Pull request by darkdef**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #4

### CODE REVIEW

File: config/console.php [hunk 1]
```diff
@@ -9,16 +9,16 @@
...
     Extractor::class => [
         '__construct()' => [
-            // Please set the following to use extractor
+            // Please set the following to use extractor. MessageReader and MessageWriter should be set to SAME MessageSource (One folder in PHP MessageSource)
             'messageReader' => static function () {
                 throw new \RuntimeException('You should configure MessageReader.');
             },
             'messageWriter' => static function () {
-                throw new \RuntimeException('You should  configure MessageWriter.');
+                throw new \RuntimeException('You should configure MessageWriter.');
```
Comment (terabytesoftw) [inline config/console.php:17]: Because keep the exception, you can set it by default.

Comment (darkdef) [reply config/console.php:17]: And if the user uses DB Source?

Comment (terabytesoftw) [reply config/console.php:17]: Well that's a point, you could have a commented configuration .

Comment (darkdef) [reply config/console.php:17]: I'm removed commented lines from config. Examples of config - in readme.md

File: README.md [hunk 2]
```diff
@@ -43,13 +43,35 @@ use \Yiisoft\Translator\Message\Php\MessageSource;
...
-            'messageReader' => static fn () => new MessageSource(dirname(__DIR__, 5) . '/messages'),
-            'messageWriter' => static fn () => new MessageSource(dirname(__DIR__, 5) . '/messages'),
+            'messageReader' => static fn () => new MessageSource($params['yiisoft/translator']['categorySources']),
+            'messageWriter' => static fn () => new MessageSource($params['yiisoft/translator']['categorySources']),
         ],
     ],
 ];
 ```
 
+And in file params.php you can config parameters of message source (if needed)
```
Comment (samdark) [inline README.md:53]:
```suggestion
And in `params.php` file you can configure parameters of a message source:
```

File: README.md [hunk 3]
```diff
@@ -43,13 +43,35 @@ use \Yiisoft\Translator\Message\Php\MessageSource;
...
+
+```php
+return [
+    'yiisoft/yii-console' => [
+        'commands' => [
+            'translator/extract' => ExtractCommand::class,
+        ],
+    ],
+    'yiisoft/translator-extractor' => [
+        // example with relative path
```
Comment (samdark) [inline README.md:63]:
```suggestion
        // Using relative path:
```

File: README.md [hunk 4]
```diff
@@ -43,13 +43,35 @@ use \Yiisoft\Translator\Message\Php\MessageSource;
...
+        'messagePath' => dirname(__DIR__, 5) . '/messages',
+         
+        // example with usage alias of application
```
Comment (samdark) [inline README.md:66]:
```suggestion
        // Usage aliases:
```

File: README.md [hunk 5]
```diff
@@ -43,13 +43,35 @@ use \Yiisoft\Translator\Message\Php\MessageSource;
...
+        // 'messagePath' => fn (Aliases $aliases) => $aliases->get('@message'), 
+    ],
+];
+```
+
+**Attention: MessageReader and MessageWriter should be configured for using SAME MessageSource (One folder in PHP MessageSource).**
+**The extractor needs this to work with existing messages.**
```
Comment (samdark) [inline README.md:73]:
```suggestion
> **Attention**: Both `MessageReader` and `MessageWriter` should be configured for using _the same_ `MessageSource`. The extractor needs it to work with existing messages.
```

File: config/console.php [hunk 6]
```diff
@@ -9,16 +9,13 @@
 return [
     Extractor::class => [
         '__construct()' => [
-            // Please set the following to use extractor
+            // Please set the following to use extractor. MessageReader and MessageWriter should be set to SAME MessageSource (One folder in PHP MessageSource)
```
Comment (samdark) [inline config/console.php:12]:
```suggestion
            // Please set the following to use extractor.
            // MessageReader and MessageWriter should be set to the SAME MessageSource.
```

**Comment by samdark** [comment]
👍

**Pull request merged**